i18n/en.json

```diff
@@ -8,5 +8,8 @@
   "exportModalIncludeRelated": "Include related documents",
   "exportModalIncludeChildren": "Include children of this page",
   "exportModalIncludeRelatedSettings": "Related Documents Settings",
-  "exportModalNoRelatedTypes": "No Related Types"
+  "exportModalNoRelatedTypes": "No Related Types",
+  "import": "Import",
+  "importModalDescription": "Importing content requires a zip file. See our <a href=\"https://v3.docs.apostrophecms.org/\" target=\"_blank\">guide to importing content</a> in Apostrophe.",
+  "importType": "Import {{ type }}"
```

Comment:
It matches the designs, just a note, the same route will be called whatever document or page is imported. 
So it might be clearer to say, `Import Documents`. We can ask @stuartromanek maybe?